### ISSUE TITLE
Disable SSL Certificate check

### ISSUE THREAD

**Issue by axtux** [ncw/rclone#168]
Hello,

I'm having some trouble with SSL certificate. Why not add an option to ignore invalid SSL certificates ? This could temporarily fix my trouble.

My trouble (maybe I should open another issue) : I'm using rclone linux 32bits binary on DNS-320L using Alt-F firmware. I can download files from my website using wget (SSL certificate is trusted) but I get an error with rclone :
`Failed to create file system for "hubic:": Get https://hubic.ga/v1.0: x509: failed to load system roots and no roots provided`

Best regards,
Axtux

**Comment by ncw** [comment]
Go tries to load the root certificates from these places on linux.

```
	"/etc/ssl/certs/ca-certificates.crt", // Debian/Ubuntu/Gentoo etc.
	"/etc/pki/tls/certs/ca-bundle.crt",   // Fedora/RHEL
	"/etc/ssl/ca-bundle.pem",             // OpenSUSE
	"/etc/pki/tls/cacert.pem",            // OpenELEC
```

Do you have something in one of those places? Or maybe somewhere else?

I wonder where wget is looking for the certs?  Maybe you could strace it and find out?

I have thought about putting an `--insecure` flag in which would ignore the certificate check, but I'm not sure it would help in the case of this error.

**Comment by ncw** [comment]
PS it might be possible to support hubic directly - rclone knows how to do oauth

**Comment by axtux** [comment]
Thanks for your fast answer. As a workaround, I took the file /etc/ssl/certs/ca-certificates.crt from my debian 8 and copied it into my NAS. Now rclone is accepting my server certificate. Not sure where wget is looking for CA but I don't have debug tools on my NAS to strace it.

The insecure flag should help for testing. In my case, I had to set a valid certificate to start testing rclone. I think should be easy to implement and would be very useful.

The gateway is just one PHP file + one SSL certificate (free with CloudFlare) https://github.com/oderwat/hubic2swiftgate and only used for authentication process, file transfer connects directly to Hubic openstack API to get full speed. As hubic authentication is not "standard", I don't see the point to implement it. The gateway is still needed for other applications that refused to implement Hubic as CyberDuck https://trac.cyberduck.io/ticket/7764 Though, if you want to implement it, Hubic API Docs can be found here https://api.hubic.com/console/ and a sandbox here https://api.hubic.com/sandbox/
